src/views/transaction/transaction.view.jsx

```diff
@@ -339,7 +343,7 @@ const mapDispatchToProps = (dispatch) => ({
   onGoToTransactionOverviewStep: (transaction) =>
     dispatch(transactionActions.goToReviewTransactionStep(transaction)),
   onFinishTransaction: (transactionType, accountIndex, redirectTo) => {
-    if (transactionType === TxType.Withdraw) {
+    if (transactionType === TxType.Withdraw || transactionType === TxType.Deposit) {
       if (redirectTo === WithdrawRedirectionRoute.Home) {
```

Comment:
I think that we don't need this check anymore right? Because the `redirectTo` variable just contains information from where the user comes from (from the `home` view or from the `account-details` view).

This is useful when the user clicks the close button to the redirect him to the view he comes from. But I think that now we don't need to take this into account after the transaction has been finished and just redirect the user to the `account-details` view if he is making a L2 transaction, and to the `home` view otherwise.

Fix me if I'm wrong Miros! Just trying to wrap my head around what we currently are doing and this new flow 😄